test/array-analysis.test.js

```diff
@@ -0,0 +1,34 @@
+const analyze = require('../src/array-analysis');
+
+const array = [1, 8, 3, 4, 2, 6];
+const obj = analyze(array);
+
+test('it analyses an array and returns an object', () => {
+  expect(analyze([1, 8, 3, 4, 2, 6])).toStrictEqual({
```

Comment:
Maybe your intention was to use `obj` here? 🤔

Reply:
Oh, I used toBe and it gave an error and jest suggested I used toStrictEqual.